### ISSUE TITLE
[BUG] unable to execute drizzle-kit cli after upgrade

### ISSUE THREAD

**Issue by woodybriggs** [drizzle-team/drizzle-kit-mirror#92]
```
drizzle-kit introspect:mysql

[redacted]/node_modules/.bin/drizzle-kit: line 16: exec: -S: invalid option
exec: usage: exec [-cl] [-a name] file [redirection ...]

```

After upgrading to 0.19.1 I am no longer to execute the drizzle-kit cli. 

Package Manager: pnpm
OS: Mac OS 13.0.1
Processor Type: Apple Silicon M1

**Comment by AndriiSherman** [comment]
Working on a fix here

https://github.com/drizzle-team/drizzle-orm/issues/781

**Comment by graemegeorge** [comment]
If it's helpful, error also occurs using for new sqlite push:
```bash
drizzle-kit push:sqlite
```
Package Manager: pnpm
OS: Mac OS 13.0.1
Processor Type: Apple Silicon M1

**Comment by AndriiSherman** [comment]
Explained a fix [here](https://github.com/drizzle-team/drizzle-orm/issues/781#issuecomment-1600679308)

will be in latest today

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.19.2`
